### PR TITLE
HAWKULAR-784 - Enable compression for serving the Hawkular UI from Undertow

### PR DESCRIPTION
**Pull request by jmazzitelli**
…dertow

### CODE REVIEW

File: dist/src/main/resources/standalone.xsl [hunk 1]
```diff
@@ -216,6 +216,21 @@
     <xsl:call-template name="loggers"/>
   </xsl:template>
 
+  <!-- Enable gzip compression when serving javascript and ccs -->
+  <xsl:template match="undertow:subsystem/undertow:server/undertow:host">
+    <xsl:copy>
+      <xsl:apply-templates select="node()|@*"/>
+      <filter-ref name="gzipFilter" predicate="path-suffix['.css'] or path-suffix['.js']"/>
```
Comment (mtho11) [inline dist/src/main/resources/standalone.xsl]: What about *.json and *.html?

Comment (jmazzitelli) [inline dist/src/main/resources/standalone.xsl]: We can add those additional path-suffix values, and anything else we deem necessary.

**Comment by mtho11** [comment]
LGTM: Looks like it took the main hawkular js codebase: hawkular-console.js from 444 Kb to 72 Kb. 
Not too shabby. :+1:

**Comment by mtho11** [comment]
Merged #820

**Pull request merged**